### DISCUSSION TITLE
Healthcheck

### DISCUSSION THREAD

**Discussion by techieferret** [zwave-js/zwave-js-ui#4015] [General]
In the docker healthcheck... It doesn't work.  docker: invalid reference format.
I tried setting the first and last " to ' and still didn't work.  Just saying the docs have errors and I don't know how to fix that.
docker run <other options here> --health-cmd="wget --no-verbose --spider --no-check-certificate --header "Accept: text/plain" https://localhost:8091/health || exit 1" --health-start-period=30s

**Comment by robertsLando** [comment]
Could you try using docker-compose instead?

**Reply by robertsLando** [reply]
https://zwave-js.github.io/zwave-js-ui/#/getting-started/docker?id=healthcheck
